src/google_takeout_metadata/statistics.py

```
@@ -0,0 +1,173 @@
+                "failed_files": self.failed_files,
+                "skipped_files": self.skipped_files,
+                "fixed_extensions": self.fixed_extensions,
+                "errors_by_type": self.errors_by_type
+            }
+        }
+        
+        try:
+            with open(log_file, 'w', encoding='utf-8') as f:
+                json.dump(report, f, indent=2, ensure_ascii=False)
+            logger.info(f"📄 Rapport détaillé sauvegardé : {log_file}")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)